### PR TITLE
Weekly #8: March 14th Sprint

### PR DESCRIPTION
**Pull request by RichardLitt**
Init.

### CODE REVIEW

**Comment by RichardLitt** [comment]
Needs:
- [x] @eminence list of commiters
- [x] @haadcode feedback
- [x] @noffle feedback
- [x] @diasdavid feedback
- [x] @jbenet feedback

File: published/008-2016-March-21.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [March 14](//github.com/ipfs/pm/issues/97) sprint:
+
+## Updates
+
+### PubSub
+
+There are some active discussions on PubSub, following a good video chat last week during the hangouts. PubSub is the name we're using to talk about a simple protocol which will help facilitate easy publishing and subscribing on top of IPFS. Our requirements are that it must be easy to implement, well-layered, and mesh well with the rest of the IFPS abstractions. To join the conversation about the PubSub API, check out this [issue](https://github.com/ipfs/notes/issues/118). For more discussions, check out all of the [PubSub issues in the notes repo](https://github.com/ipfs/notes/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+pubsub).
```
Comment (yuvipanda) [inline published/008-2016-March-21.md]: 'IFPS' -> 'IPFS'

Comment (RichardLitt) [inline published/008-2016-March-21.md]: Thanks!

**Comment by jbenet** [comment]
@RichardLitt stuff there LGTM. i think in all the contributions there's probably stuff to highlight? 
- maybe add @noffle's wiki?

Ideas for more content:
- @whyrusleeping and @cryptix mention the git-ipfs tools to rehost repos
- @dignifiedquire gives a preview of upcoming webui changes

**Comment by haadcode** [comment]
Can we add something really small about v0.4, "it's coming very soon, almost done testing" or smth?

**Comment by jbenet** [comment]
@haadcode yes -- let's add a note about the stress testing @whyrusleeping did.

**Comment by jbenet** [comment]
@diasdavid anything more to highlight here on your end?

**Comment by RichardLitt** [comment]
@jbenet I asked people specifically on the hangouts, and these were the only large takeaways that I could identify. :+1: for addin stress test and note about 0.4.0. @dignifiedquire Want to throw me some images or vids for webui?

**Comment by jbenet** [comment]
images or gifs-- should be renderable on email. (can link out to a video, and should have an img or gif to embed it with)

**Comment by hackergrrl** [comment]
I think ip-wiki isn't quite ready to share yet. I'd like to hold off until I have a working demo.

**Comment by jbenet** [comment]
@RichardLitt worth highlighting recent updates to this: https://groups.google.com/forum/#!topic/ipfs-users/ERhgN8BMzhY

**Comment by lidel** [comment]
See my comment in  https://github.com/ipfs/weekly/issues/29#issuecomment-199527414, feel free to copy the note about Firefox addon.

**Comment by dignifiedquire** [comment]
Current version of the files explorer:

-.-

**Comment by RichardLitt** [comment]
@dignifiedquire Looks good. How about:

### [webui](//github.com/ipfs/webui)

[@dignifiedquire](//github.com/dignifiedquire) has been working hard on the new webui, which will be coming soon. You'll be able to preview images, watch videos, create folders, drag and drop, and more. Here's a pretty gif. To help out, check out the [help wanted labels](https://github.com/ipfs/webui/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) on the repo.

**Comment by daviddias** [comment]
I've reviewed and looked at my list of things. Everything from my sprint that was exciting went on the last weekly, now it is just pain from all the debugging nasty bugs.

**Pull request merged**